src/rviz/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ if(NEW_YAMLCPP_FOUND)
   add_definitions(-DRVIZ_HAVE_YAMLCPP_05)
 endif(NEW_YAMLCPP_FOUND)
 
+find_package(X11 REQUIRED)
```

Comment:
This needs to be wrapped in a `if(UNIX AND NOT APPLE)` block, as the `find_package(X11 ...)` is not required on OS X.